kerml/src/examples/Behavior Examples/TakePicture.kerml

```diff
@@ -12,7 +12,7 @@ behavior TakePicture() {
 	step step1: Focus[1] (out xrsl);	
 	step step2: Shoot[1] (in xsf);
 	
-	flow exposure[1] from step1.xrsl to step2.xsf;
+	succession flow exposure[1] from step1.xrsl to step2.xsf;
```

Comment:
This has nothing to do with the current PR, but just a question for my understanding: is multiplicity [1] on Focus, Shoot and exposure mandatory here? Or would they be the default when omitted?

Reply:
KerML does not specify any special multiplicity defaults. Unless otherwise give explicitly, the multiplicity is as inherited from the implicit base feature element, which is 0..*. In this case, though, even if the model was done in SysML, with actions instead of steps, the default multiplicity would still be 0..*. That is because a behavioral feature is supposed, by default, to be able to be performed zero or more times, unless otherwise constrained (and a flow is also a behavioral feature, as well as a connection).